src/quo2/components/navigation/page_nav.cljs

```diff
@@ -30,15 +30,16 @@
            {:no-color true})))
 
 (defn left-section-view
-  [{:keys [on-press icon accessibility-label type] :or {type :grey}}
+  [{:keys [on-press icon accessibility-label type icon-override-theme] :or {type :grey}}
    put-middle-section-on-left?]
   [rn/view {:style (when put-middle-section-on-left? {:margin-right 5})}
    [button/button
     {:on-press            on-press
      :icon                true
      :type                type
      :size                32
-     :accessibility-label accessibility-label}
+     :accessibility-label accessibility-label
+     :override-theme      icon-override-theme}
```

Comment:
why not just override-theme? 🤔

Reply:
Because it's specific for the icon, it's a replacement for the old (and not working) `:icon-color` & `:icon-background-color` parameters.